File: Height Checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> new_heights = heights;
        sort(new_heights.begin(), new_heights.end());
        int count = 0;
        for (int i = 0; i < heights.size(); i++) {
            count += (new_heights[i] != heights[i]);
        }
        return count;
    }
};

File: Height Checker.go
func heightChecker(heights []int) int {
    new_heights := append([]int{}, heights...)
    sort.Ints(new_heights)
    count := 0
    for i := 0; i < len(heights); i++ {
        if (heights[i] != new_heights[i]) {
            count++
        }
    }
    return count
}
